core/chains/evm/config/toml/defaults/Scroll_Mainnet.toml

```diff
@@ -1,20 +1,22 @@
 ChainID = '534352'
 FinalityDepth = 10
+FinalityTagEnabled = true
 ChainType = 'scroll'
-LogPollInterval = '3s'
+LogPollInterval = '5s'
```

Comment:
Im actually not sure if this has any impact

Reply:
the soak test(s) we ran revealed issues with log polling adjusting this value fixed them

Reply:
I see thats interesting, can you please point me to what the issue was, It could help in the future

Reply:
yeah was receiving consistent warnings throughout the soak about logs not being to be able to be fetched from the block. after we adjusted this interval the issue went away

Reply:
Also adding that the upgrade includes a change to not fix the block rate at 3s. So I think this change might help give a bit of tolerance to fluctuations in the block rate.